source/docs/installation.md

```diff
@@ -0,0 +1,66 @@
+title: Installation
+---
+
+## Prerequisites
+
+In order to make Embark work on our machine, we need to have some tools installed first. Make sure you have the following ready and in the correct version:
+
+- [Node](#Node)
+- [IPFS](#IPFS-Optional)
+- [Ethereum Client](#Ethereum-Client-Optional)
+
+Once done, go ahead and [install Embark](#Installing-Embark).
+
+### Node
+
+Please install [Node.js](http://nodejs.org/) in version 8.11.3 LTS or higher.
+
+{% note info Quick tip: %}
+**We recommend installing Node using the [Node Version Manager](https://github.com/creationix/nvm/blob/master/README.md).**  This is because it makes it very easy to install different versions of Node in isolated environments that don't require users to [change their permissions](https://docs.npmjs.com/getting-started/fixing-npm-permissions) when installing packages. Find instructions on how to install NVM [here](https://github.com/creationix/nvm/blob/master/README.md#install-script).
+
+Once that is done, you can install and select a specific Node version using the `nvm` command like this:
+
+<pre><code class="shell">$ nvm install 8.11
+$ nvm use 8.11
+</code></pre>
+
+{% endnote %}
+
+### IPFS (Optional)
+
+IPFS can be used to distribute your application's content on the decentralized IPFS nodes. This can be skipped in case this isn't planned, however we do recommend it. Checkout IPFS' [installation guide](https://docs.ipfs.io/introduction/install/) to learn how to install IPFS on your local machine.
+
+To verify that the installation was successful, simply run the following command:
+
+<pre><code class="shell">$ ipfs --version
+</code></pre>
+
+### Ethereum Client (Optional)
+
+Embark can spin up an Ethereum node for you. To make this happen, an Ethereum client has to be installed on your machine as well. There's several clients to choose from.
+
+[Ganache CLI](https://truffleframework.com/ganache) is probably the best client for development purposes as it emulates a blockchain and is therefore fast than a real node:
+
+<pre><code class="shell">$ npm -g install ganache-cli</code></pre>
+
+In case you want to run a real node, [geth](https://geth.ethereum.org/) is a pretty good one. Check out the [installation guide](https://ethereum.github.io/go-ethereum/install/) for your platform and verify your installation with:
+
+<pre><code class="shell">$ geth --version</code></pre>
```

Comment:
I thought it's better to just refer to the existing installation instructions from geth so we don't have to maintain those. Let me know if this should be changed.

Reply:
Can we include the output from the tool? I want to see what to look for when I run this command.

Reply:
Absolutely. Let's add this.